plugins/amazonq/chat/jetbrains-community/resources/META-INF/plugin-chat.xml

```diff
@@ -29,6 +29,10 @@
 
     <actions>
         <action id="aws.toolkit.open.q.window" class="software.aws.toolkits.jetbrains.services.amazonq.QRefreshPanelAction"/>
+        <action id="aws.toolkit.jetbrains.core.services.cwc.inline.openChat" class="software.aws.toolkits.jetbrains.services.cwc.inline.OpenChatInputAction">
+            <keyboard-shortcut first-keystroke="meta I" keymap="$default"/>
```

Comment:
is one of these supposed to be mac only

Reply:
I think it's meta for mac and linux and ctrl for windows

Reply:
please filter accordingly so they only apply to the relevant OS

https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html#idea-plugin__actions__action__keyboard-shortcut